database/migrations/2017_12_19_210400_add_editors_status.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class AddEditorsStatus extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
```

Comment:
we should think about making a flags table where we can register flags for stuff, e.g:
user1 flagged post10
user1 flagged post20
user2 flagged post10

This way we can see and sort by amount of flags so we can identity bad posts/comments faster and more easily

Reply:
@timothywalter the flags were alle ready register by flags.